cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
+
+ADD . /go/src/github.com/uber/jaeger
+
+ENTRYPOINT go run /go/src/github.com/uber/jaeger/cmd/standalone/main.go --span-storage.type=memory
+
+EXPOSE 5775 6831 6832 14267 16686
```

Comment:
I can't find this comment. Our port definitions are all over the place, including work that was done in the agent. Is this something we want to tackle in this PR or is this a request that should be encapsulated in a separate ticket?